### PR TITLE
DDF-2881 added to configuring template section

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates managing section of jbake-templated docs with Configuring sections.

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.
@clockard
@coyotesqrl
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- verify headers added to .adoc files (Contents pre-existing, review of content optional)
- verify templates (.ftl files)
- verify all relevant sections being included (pdf attached, compare with current documentation.pdf)

#### What are the relevant tickets?

[DDF-2881](https://codice.atlassian.net/browse/DDF-2881)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/ddf/files/1046074/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/jdocs/content/_configuring/admin-console-intro-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:title: Admin Console Intro
+:type: configuringAdminConsole
+:status: published
+:summary: Introduction to the ${admin-console}
```
Comment (mcalcote) [inline distribution/docs/src/main/jdocs/content/_configuring/admin-console-intro-contents.adoc]: A few of these summaries are missing periods

File: distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+:title: Managing Applications From the ${command-console}
+:type: configuringCommandConsole
+:status: published
+:summary: Managing Applications From the ${command-console}.
```
Comment (mcalcote) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc]: Some of the summaries, especially those with the same text as the title have words starting in (mostly) caps, but others only have the first word in caps, like a normal sentence.

File: distribution/docs/src/main/jdocs/content/_configuring/admin-console-intro-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/admin-console-intro-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc]: They should be sentence case. Some of them reference specific modules/components, which I've capitalized. Going to go back through and make sure they are clear and consistent.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
+:title: Admin Console Tutorial
+:type: configuringAdminConsole
+:status: published
+:summary: Tutorial for the ${admin-console}.
+:order: 01
+
+.${admin-console} Tutorial
+****
+This overview covers general uses for the ${admin-console}.
```
Comment (shaundmorris) [inline distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc]: ❓ This section seems to have a background down to the end of the page at `Disabled apps will be moved to a list at the bottom of the`. Is that intended?

File: distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
+:title: Hardening Intro
+:type: configuringIntro
+:status: published
+:summary: Introduction to Hardening.
+:order: 03
+
+.Security Hardening
+****
```
Comment (shaundmorris) [inline distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc]: ❓ should more of these sections have this background? I see it on `Security Hardening` and `Admin Console Tutorial`?

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-landing-page-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
+:title: Configuring Landing Page
+:type: configuringAdminConsole
+:status: published
+:summary: Configure landing page.
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-landing-page-contents.adoc]: I'm not sure. It seems a little grandiose to me, but I'm open to changing it.

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc [hunk 6]
```diff
@@ -0,0 +1,82 @@
+:title: Configuring the Web Context Policy Manager
+:type: configuringAdminConsole
+:status: published
+:summary: Configuring the web context policy manager.
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-context-policy-manager-contents.adoc]: I wasn't sure, but I am seeing we do capitalize it elsewhere.

File: distribution/docs/src/main/jdocs/content/_configuring/standalone-sts-contents.adoc [hunk 7]
```diff
@@ -0,0 +1,17 @@
+:title: Standalone Security Token Service (STS) Installation
+:type: configuringCommandConsole
+:status: published
+:summary: Standalone Security Token Service (STS) installation.
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/standalone-sts-contents.adoc]: agreed

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc [hunk 8]
```diff
@@ -0,0 +1,12 @@
+:title: Configuring Web Service Providers
+:type: configuringConfigFile
+:status: published
+:summary: Configuring web service providers.
```
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc]: Updated.

File: distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc]: I wanted to use the different formatting to offset certain sections visually, but the pdf version doesn't handle it well, so I'll be removing it.

File: distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/hardening-intro-contents.adoc]: I was going to start using it more, but am now finding that it doesn't translate well to pdf if there are page breaks.

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/jdocs/content/_configuring/command-console-intro-contents.adoc [hunk 9]
```diff
@@ -0,0 +1,7 @@
+:title: ${command-console} Intro
+:type: configuringCommandConsole
+:status: published
+:summary: ${command-console} Intro.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/command-console-intro-contents.adoc]: So far the summaries have only capitalized the first word but here "Intro" is capitalized as well.

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-catalog-authorization-contents.adoc [hunk 10]
```diff
@@ -0,0 +1,25 @@
+:title: Configuring Catalog Operation Authorization
+:type: configuringAdminConsole
+:status: published
+:summary: Configuring Catalog Operation Authorization.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-catalog-authorization-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-intro-contents.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
+:title: Configuring Intro
+:type: configuringIntro
+:status: published
+:summary: Introduction to Configuring.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-intro-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc [hunk 12]
```diff
@@ -0,0 +1,12 @@
+:title: Configuring Web Service Providers
+:type: configuringConfigFile
+:status: published
+:summary: Configuring Web Service Providers.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/installing-multiple-installations-contents.adoc [hunk 13]
```diff
@@ -0,0 +1,43 @@
+:title: Installing Multiple ${branding}s on the Same Host
+:type: configuringConfigFile
+:status: published
+:summary: Installing Multiple ${branding}s on the Same Host.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/installing-multiple-installations-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc [hunk 14]
```diff
@@ -0,0 +1,316 @@
+:title: Managing Keystores and Certificates
+:type: configuringIntro
+:status: published
+:summary: Keystore and Certificate instructions.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/securing-intro-contents.adoc [hunk 15]
```diff
@@ -0,0 +1,19 @@
+:title: Securing Intro
+:type: configuringIntro
+:status: published
+:summary: Introduction to Securing.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_configuring/securing-intro-contents.adoc]: capitalization?

File: distribution/docs/src/main/jdocs/content/_configuring/command-console-intro-contents.adoc (hunk 9 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/command-console-intro-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-catalog-authorization-contents.adoc (hunk 10 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-catalog-authorization-contents.adoc]: fixed...

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-intro-contents.adoc (hunk 11 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-intro-contents.adoc]: reworded slightly.

File: distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc (hunk 12 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc]: I squashed the commit so I can't find the comment, but I think someone suggested that Web Service Providers should be capitalized.

File: distribution/docs/src/main/jdocs/content/_configuring/installing-multiple-installations-contents.adoc (hunk 13 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/installing-multiple-installations-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc (hunk 14 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/managing-certificates-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_configuring/securing-intro-contents.adoc (hunk 15 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_configuring/securing-intro-contents.adoc]: reworded and fixed.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**